src/http.ts

```diff
@@ -1359,6 +1359,13 @@ export class LemmyHttp {
       return json;
     }
   }
+
+  /**
+   * Set the headers (can be used to set the auth header)
+   */
+  setHeaders(headers: { [key: string]: string }) {
+    this.#headers = headers;
+  }
```

Comment:
Should consumers of the client also be able to pass headers for individual requests?

Reply:
I can't think of a case at least. But this one seems useful, because a common case is to Login, then set the auth header ( This might not be necessary in browsers where it *should* pass the cookie, but in other clients its seems useful).